src/AsyncTCP.cpp

```diff
@@ -29,7 +29,9 @@ extern "C"{
 #include "lwip/dns.h"
 #include "lwip/err.h"
 }
+#if CONFIG_ASYNC_TCP_USE_WDT
```

Comment:
[In AsyncTCP.h](https://github.com/esphome/AsyncTCP/blob/master/src/AsyncTCP.h#L36) - WDT is used only if `CONFIG_ASYNC_TCP_USE_WDT` is set, so that #if just removes the unneeded include (which is obviously not available in LT).